# D365FO-ManagedExceptions

It's a proof of concept showing how managed (.NET, CLR) exceptions can be thrown from X++ code in Microsoft Dynamics 365 for Finance and Operations.

Refer to the blog post for more details. (The blog post will be published soon and then a link will be added to this page.)

To run the code, you must put the content of the *Metadata* folder to the modelstore of D365FO (e.g. *C:\AOSService\PackagesLocalDirectory* or *K:\AOSService\PackagesLocalDirectory*).

Code was built on platform update 20.
